HumanControls/DisplayController.cpp

```diff
@@ -21,4 +27,25 @@ void DisplayController::printRegion(int x, int y, String text)
 {
     this->m_lcd.setCursor(x, y);
     this->m_lcd.print(text);
+}
+
+void DisplayController::printRegion(int x, int y, int code)
+{
+    this->m_lcd.setCursor(x, y);
+    this->m_lcd.write(code);
+}
+
+void DisplayController::printBattery(int x, int y, int code, double level)
```

Comment:
If the custom code for a battery is the same each time, maybe just put that in a constant?

Reply:
We cannot, battery level changes makes the battery custom char change.